### PR TITLE
remove splash page content from repo

### PR DESCRIPTION
**Pull request by JessicaS11**
The template this repo was based on had a separate splash webpage independent of the JupyterBook. We'll be using the webpage built with godaddy for our domain, so this PR removes the actions and content associated with the splash page.

### CODE REVIEW

File: conda/environment.yml [hunk 1]
```diff
@@ -2,12 +2,11 @@
...
 # !!! If you are developing the website locally use the existing lock files
-# for your platform (`conda create --name hackweek --file conda-osx-64.lock`)
-name: hackweek
+# for your platform (`conda create --name cryocloud --file conda-linux-64.lock`)
+name: cryocloud
 channels:
   - conda-forge
-dependencies:
-  - cookiecutter
+dependencies:
```
Comment (weiji14) [inline conda/environment.yml:9]: Extra whitespace removal
```suggestion
dependencies:
```

File: conda/environment.yml [hunk 2]
```diff
@@ -20,10 +19,5 @@ dependencies:
...
   - vim
   - wget
-  - jinja2 >= 2.11
-  - pygments>=2.6.1
-  - pymdown-extensions>=7.1
-  - pip:
-    - jinja-markdown==1.210911
 platforms:
-  - linux-64
+  - linux-64
```
Comment (weiji14) [inline conda/environment.yml:23]:
```suggestion
  - linux-64

```

File: conda/lock-environment.sh [hunk 3]
```diff
@@ -35,4 +36,4 @@ conda-lock render -k env
 if [[ $? -eq 0 ]]; then
   conda deactivate
   conda remove -q -y --name ${LOCK_ENV} --all
-fi
+fi
```
Comment (weiji14) [inline conda/lock-environment.sh:39]:
```suggestion
fi

```

File: scripts/README.md [hunk 4]
```diff
@@ -3,7 +3,6 @@
 This folder contains several scripts for building our splash page and Jupyter Book website. At a high level, human-friendly configuration files are read and translated into HTML to post to a static website hosting server.
 
 * `build_resources.sh`
-  1. reads `cookiecutter.yaml` and runs `cookiecutter` to insert variables into the splash page HTML
-  2. reads Jupyter Book files under the `book/` directory and runs `jupyter book build` to render the Jupyter Book part of the website.
-
-* `yaml_tools.py` and `yaml2json.py` are helper scripts to create intermediate `json` files that are required by `cookiecutter`
+  1. reads Jupyter Book files under the `book/` directory and runs `jupyter book build` to render the Jupyter Book part of the website.
+  Note: This script previously also handled (with a few helper scripts) the building of a splash page.
+  See the [template repository](https://github.com/uwhackweek/jupyterbook-template) for the original functionlity.
```
Comment (weiji14) [inline scripts/README.md:8]: Typo
```suggestion
  See the [template repository](https://github.com/uwhackweek/jupyterbook-template) for the original functionality.
```

**Review by weiji14 — COMMENTED** [review]
A few minor typos. The spellcheck workflow at https://github.com/CryoInTheCloud/CryoCloudWebsite/actions/runs/3268735537/jobs/5375483610#step:7:47 seems to be complaining about something, but I couldn't quite understand why it's looking at files under `.git/objects/*` :sweat_smile:

File: conda/lock-environment.sh (hunk 3 above)
Comment (JessicaS11) [reply conda/lock-environment.sh:39]: are the blank newlines important for running?

Comment (weiji14) [reply conda/lock-environment.sh]: Not really, it's just good to be consistent (have all files ending with a newline) for diff-ing. See https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file.

image

**Comment by JessicaS11** [comment]
> I couldn't quite understand why it's looking at files under `.git/objects/*`

it's because `check_hidden: true` is set... I'm not sure why it needs to be true (are there hidden jupyterbook or notebook files we'd want reviewed)? Otherwise I can set it to false...

**Comment by tsnow03** [comment]
I think you can set to False and then we can change it back if an issue arises. I haven't seen any hidden notebooks, but we can keep an eye out and revisit if needed.

**Comment by JessicaS11** [comment]
there are also a bunch of encoding issues I've never seen before that Codespell is throwing warnings for. Maybe something in one of our [local git setups](https://stackoverflow.com/questions/58352957/what-going-on-git-see-changes-without-changes)? 🤷‍♀️

**Pull request merged**